### DISCUSSION TITLE
[Feature] Automatic background service upload should have slider to delete uploaded pictures

### DISCUSSION THREAD

**Discussion by ckocyigit** [immich-app/immich#2420] [Feature Request]
### The feature

Automatic background service upload should have slider to delete uploaded pictures

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #165

**Reply by ckocyigit** [reply]
I see this Feature request as a different Feature to #165 tbh.

#165 Suggests a batch/bulk-like post-upload command to free up space which can happen async.

This slider however would make it possible to delete local media while uploading guaranteeing a backup on immich and a mess-free phone.

Also I would guess that this could only impact mobile code - a simple upload and delete afterwards process.

**Reply by ckocyigit** [reply]
![example](https://github.com/immich-app/immich/assets/60385015/5d50eefc-740c-41ba-93c7-6fbcdfa151ad)

**Reply by ckocyigit** [reply]
@bo0tzz is this still a duplicate ?
